components/composer/src/Composer.svelte

```diff
@@ -347,7 +347,7 @@
       // If custom url supplied
       themeUrl = theme;
     } else if (theme) {
-      themeUrl = `https://unpkg.com/@nylas/components-composer@${pkg.version}/themes/${theme}.css`;
+      themeUrl = `../themes/${theme}.css`;
```

Comment:
Using the local file is better for testing, but not sure if it will break in prod? cc @ozsivanov

Reply:
not sure about this one, can you make sure the file is included in the output bundle when you build?